docs/networking/diagnostics/diagnosing-network-issues-with-mtr/index.md

```diff
@@ -20,13 +20,13 @@ external_resources:
 
 ![Diagnosing Network Issues with MTR](diagnosing-network-issues-with-mtr.png)
 
-[MTR](http://www.bitwizard.nl/mtr/) is a powerful tool which enables administrators to diagnose and isolate networking errors and provide reports of network status to upstream providers. MTR represents an evolution of the `traceroute` command by providing a greater data sample, as if augmenting `traceroute` with `ping` output. This document provides an in depth overview of MTR, the data it generates, and how to interpret and draw conclusions based on the data provided by it.
+[MTR](http://www.bitwizard.nl/mtr/) is a powerful tool that enables administrators to diagnose and isolate networking errors and provide reports of network status to upstream providers. MTR represents an evolution of the `traceroute` command by providing a greater data sample as if augmenting `traceroute` with `ping` output. This document provides an in-depth overview of MTR, the data it generates, and how to interpret and draw conclusions based on the data provided by it.
 
 For a basic overview of network diagnostic techniques, see our introduction to [network diagnostics](/docs/tools-reference/linux-system-administration-basics/#network-diagnostics). If you are having general issues with your system, read our overview of general [system diagnostics](/docs/using-linux/administration-basics/#system-diagnostics).
 
 ## Network Diagnostics Background
 
-Networking diagnostic tools including `ping`, `traceroute`, and `mtr` use Internet Control Message Protocol (ICMP) packets to test contention and traffic between two points on the Internet. When a user pings a host on the Internet, a series of ICMP packets are sent to the host, which responds by sending packets in return. The user's client is then able to compute the round trip time between two points on the Internet.
+Networking diagnostic tools including `ping`, `traceroute`, and `mtr` use Internet Control Message Protocol (ICMP) packets to test contention and traffic between two points on the Internet. When a user pings a host on the Internet, a series of ICMP packets are sent to the host which in turn responds by sending packets in return. The user's client is then able to compute the round trip time between two points on the Internet.
```

Comment:
This isn't quite right.

Grammarly objects to `which responds` -- and I sympathize.
But, `in turn`...`in return` is awkward.